lib/ae_mdw/db/name.ex

```diff
@@ -118,33 +115,10 @@ defmodule AeMdw.Db.Name do
   @spec pointers(state(), Model.name()) :: map()
   def pointers(_state, Model.name(updates: [])), do: %{}
 
-  def pointers(
-        state,
-        Model.name(index: plain_name, updates: [{_block_index, txi} | _rest_updates])
-      ) do
-    Model.tx(id: tx_hash) = DbUtil.read_tx!(state, txi)
-    {:ok, name_hash} = :aens.get_name_hash(plain_name)
-
-    pointers =
-      case AE.Db.get_tx_data(tx_hash) do
-        {_block_hash, :name_update_tx, _signed_tx, tx_rec} ->
-          :aens_update_tx.pointers(tx_rec)
-
-        {_block_hash, :contract_call_tx, _signed_tx, _tx_rec} ->
-          state
-          |> Contracts.fetch_int_contract_calls(txi, "AENS.update")
-          |> Stream.map(fn Model.int_contract_call(tx: aetx) ->
-            {:name_update_tx, tx} = :aetx.specialize_type(aetx)
-
-            tx
-          end)
-          |> Enum.find(fn tx ->
-            name_hash == :aens_update_tx.name_hash(tx)
-          end)
-          |> :aens_update_tx.pointers()
-      end
-
-    pointers
+  def pointers(state, Model.name(updates: [{_block_index, txi_idx} | _rest_updates])) do
```

Comment:
Could we take the opportunity to extend the test cases to cover the internal calls and `paying_for` (for `ga_meta` would be nice too but we have on `name_controller_test` a thorough one)?